test/cypress/e2e/admin-prompt-entry.cy.js

```diff
@@ -72,31 +76,51 @@ describe('Admin Prompt & Entry', () => {
     cy.get('[data-test="file-image"]').selectFile('src/assets/cypress.jpg')
 
     // Get the submit button and click it
-    cy.get('[data-test="button-submit"]').click({ timeout: 5000 }).wait(2000)
+    cy.get('[data-test="button-submit"]').click()
+
+    // Check the Entry is submitted successfully
+    cy.get('.q-notification__message').contains('Entry successfully submitted')
   })
 
   it('Should delete the entry', () => {
     // Get the second button (Delete Entry) and click it
     cy.get('[data-test="input-search"]').type('TESTER').wait(1000)
 
     // Get the expand button and click it
-    cy.get('[data-test="button-expand"]').click({ timeout: 1000 })
+    cy.get('[data-test="button-expand"]').click()
+
+    // Delete all entry in a prompt and left one
+    cy.get('[data-test="button-delete-entry"]').then(($btn)=>{
+      for(let i=$btn.length-1; i>0; i--){
+        cy.get('[data-test="button-delete-entry"]').eq(i).click({force:true})
```

Comment:
Everything else I understood but I couldn't figure out why the ```{force:true}``` was necessary?

Reply:
- Force true because if we have many entries we need to scroll. And this is just a shortcut to avoid using the scroll function
- I duplicate the code because. If we delete the last one in the loop, that means the parent will be wrong: 
   - If there is no entry line 93 will throw an error. 
   - so I delete the last one after.